### ISSUE TITLE
Check for updates periodically, not just on startup

### ISSUE THREAD

**Issue by 1j01** [1j01/tracky-mouse#84]
As this is accessibility software for basic computer access, it should be expected to be long-lived, with uptimes of over a week in some cases.
Right now it checks for updates only on startup. We should extend this to also check daily (or so).
It should probably *not* show the update prompt for the same version if it's not updated by the next check time.
